metadata.rb

```diff
@@ -13,8 +12,8 @@
 supports 'scientific'
 supports 'oracle'
 
-depends  'poise-python', '>= 1.5'
+depends  'pyenv', '~> 3.1'
 
 source_url 'https://github.com/sous-chefs/graphite'
 issues_url 'https://github.com/sous-chefs/graphite/issues'
-chef_version '>= 13'
+chef_version '>= 14'
```

Comment:
If we’re bumping the version we should bump it to the current supported Chef version (15) as a preference

Reply:
this is a selfish reason but at `$JOB` we haven't figured out how to handling the licensing for chef 15 yet so I'm stuck on 14. If thats really necessary here I'll obvious set it to 15 but I'll leave that up to the sous-chefs

Reply:
If 14 works just keep it there. It makes the client upgrades easier for users when they can use a single cookbooks against a mixed fleet during their upgrade